### ISSUE TITLE
Skip to first required field in the contact container

### ISSUE THREAD

**Issue by cagulas** [OpenSRP/opensrp-client-anc#260]
Right now, when you open a contact container, you land on the first section. From the user standpoint, it would be helpful if the app skips to the first un-answered required field in the contact container, so that they don't have to go through each section to look for it. This is important mainly for when the user is skipping around between sections and then is looking for the remaining required fields to answer before she can finalize the contact.

Low priority for now.

**Comment by cafootitt** [comment]
@dubdabasoduba I don't think I can test this, because the first required field inside each contact container's first page doesn't require scroll, so the behavior is no different from if you just open the container anyway. I'm going to move this issue back to To Do to indicate we need to finish implementing this, but label it blocked by the native form performance issue, as we discussed.

**Comment by cafootitt** [comment]
Blocked by https://github.com/OpenSRP/opensrp-client-anc/issues/218

**Comment by cafootitt** [comment]
Moving this to the top of the list, as this feature was requested again by the digital health consultant in Rwanda as he was testing for the first time.
